### DISCUSSION TITLE
LDA Differences?

### DISCUSSION THREAD

**Discussion by SeereouslyDrewNichols** [vdemichev/DiaNN#999] [General]
Hi Vadim,

Long time listener, first time caller.  When I analyze my data, I tend to do it w/ high verbosity.  I noticed that between DIANN 1.8 and 1.8.1, there are some subtle differences, particularly w/ the LDA during iRT estimation.  I have a couple of questions.

1.  I noticed now instead of 4 features used to train the LDA, there are six.  Are they the same four features w/ two additional ones? 

2. The values of the weights for a given file seem vastly different between the two versions, particularly considering that there are two additional features used.  i.e., in 1.8, many of the weight values were close to 1.  For 1.8.1, it looks like the first feature dominates the LDA as it goes as high or higher than 4 while the rest are relatively lower.  Is feature 1 the same in both versions or was there a change to either of the classifiers used? 

Thank you for your time.  I'm just trying to understand how the algorithm works.

**Comment by vdemichev** [comment]
Hi Drew,

Yes, the output is changing between versions. Primarily, this is meant to help DIA-NN development as well as diagnose problems, i.e. when someone analyses a file and does not get IDs, they can share the log with me and it can sometimes help figure out what's wrong.

1. If I recall correctly, the features are different.
2. Yes. Also, LDA is not super-stable in terms of weights, i.e. it's stable in terms of good outputs but a small change in the algorithm can lead to signficantly different weights.

Best,
Vadim

**Reply by SeereouslyDrewNichols** [reply]
Thank you Vadim!!! Are you free to discuss what the new features are? If not, I totally understand.

**Reply by vdemichev** [reply]
There's nothing fundamentally 'novel', just more fancy ways to calculate things. For example, a correlation between two XICs can be calculated using different RT windows around the apex, and the XICs themselves can be extracted using different mass tolerances - DIA-NN nowdays aims to maximise the use of information encoded in the data, hence more scores.
